functions/src/services/teamwork.service.ts

```diff
@@ -65,6 +65,46 @@ export class TeamworkService {
     }
   }
 
+  async getTaskLists(token: string, projectId: string, page: number) {
+    let authData;
+    try {
+      authData = await this.validateToken(token);
```

Comment:
Similar to the previous comment, a GET on `connections/:connectionId/projects/:projectId/task-lists`
should have enough data in the request handler to provide token/projectId to this method.